tests/ui/implicit_hasher.rs

```diff
@@ -85,19 +82,19 @@ macro_rules! gen {
     };
 
     (fn $name:ident) => {
-        pub fn $name(_map: &mut HashMap<i32, i32>, _set: &mut HashSet<i32>) {
-        }
-    }
+        pub fn $name(_map: &mut HashMap<i32, i32>, _set: &mut HashSet<i32>) {}
+    };
 }
 
-gen!(impl);
+gen!(impl );
```

Comment:
This looks like a bug in rustfmt? Might be good to make a small example out of it and report it.

Reply:
ticket: https://github.com/rust-lang/rustfmt/issues/3051